Project.toml

```diff
@@ -26,6 +26,3 @@ julia = "1"
 Crayons = "a8cc5b0e-0ffa-5ad4-8c14-923d3ee1735f"
 SymPy = "24249f21-da20-56a4-8eb1-6a02cf4ae2e6"
 Test = "8dfed614-e22c-5e08-85e1-65c5234f0b40"
-
-[targets]
```

Comment:
@janlisse:  I am not sure what happened here.  I did not intentionally remove this but also do not know what these lines do